mne/io/eeglab/eeglab.py

```diff
@@ -258,6 +306,24 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
 
         last_samps = [eeg.pnts - 1]
         info = _get_info(eeg, montage, eog=eog)
+
+        n_chan = len(info["chs"])
+        stim_chan = dict(ch_name='STI 014', coil_type=FIFF.FIFFV_COIL_NONE,
+                         kind=FIFF.FIFFV_STIM_CH, logno=n_chan + 1,
+                         scanno=n_chan + 1, cal=1., range=1., loc=np.zeros(12),
+                         unit=FIFF.FIFF_UNIT_NONE, unit_mul=0.,
+                         coord_frame=FIFF.FIFFV_COORD_UNKNOWN)
+        info['chs'].append(stim_chan)
+        if event_id is None:
+            event_id = dict()
+
+        self.preload = False  # so the event-setting works
```

Comment:
This is not nice ... it does not make sense to change a parameter that is specified differently by the user. What sense does it then make to have the preload flag?